slamd/discovery/processing/experiment/experiment_model.py

```diff
@@ -1,6 +1,22 @@
 from enum import Enum
+import os
 
 
 class ExperimentModel(Enum):
-    RANDOM_FOREST = 'AI Model (lolo Random Forest)'
     GAUSSIAN_PROCESS = 'Statistics-based model (Gaussian Process Regression)'
+    RANDOM_FOREST = 'AI Model (lolo Random Forest)'
+    PCA_GAUSSIAN_PROCESS = 'Gaussian Process with PCA'
+    PCA_RANDOM_FOREST = 'Random Forest with PCA'
+    TUNED_GAUSSIAN_PROCESS = 'Tuned Gaussian Process'
+    TUNED_RANDOM_FOREST = 'Tuned Random Forest'
+
+    @classmethod
+    def get_all_models(cls):
+        if os.getenv('FLASK_ENV') != 'demo':
```

Comment:
This works, but it might be better to turn the logic around and say if os.getenv('FLASK_ENV') == 'development'. This way, also when Christoph adds a new profile / heroku instance e.g. 'production' it is still only enabled locally and not on prod. The way it is implemented right now would lead to the effect that for a new instance / environment the tuned versions are automatically enabled even though we always only want them locally.